Makefile

```diff
@@ -6,15 +6,32 @@ CROSS ?= linux/arm linux/arm64 linux/ppc64le linux/s390x \
 SUDO ?= sudo -n
 
 .PHONY: all
-all: lint test cross
+all: clean lint test test-local cross
+
+.PHONY: clean
+clean:
+	$(RM) mount/go-local.*
 
 .PHONY: test
 test: RUN_VIA_SUDO = $(shell $(SUDO) true && echo -exec \"$(SUDO)\")
-test:
+test: clean
 	for p in $(PACKAGES); do \
 		(cd $$p && go test $(RUN_VIA_SUDO) -v .); \
 	done
 
+# test the mount module against the local mountinfo source code instead of the
+# release specified in go.mod. This allows catching regressions / breaking
+# changes in mountinfo.
+.PHONY: test-local
+test-local: RUN_VIA_SUDO = $(shell $(SUDO) true && echo -exec \"$(SUDO)\")
+test-local: clean
+	echo 'replace github.com/moby/sys/mountinfo => ../mountinfo' | cat mount/go.mod - > mount/go-local.mod \
```

Comment:
Here continuation (`\` and `&&` on the next line) is not needed. Make executes the commands via shell one by one, if a command fails, it stops.

The continuation is needed after `cd` as it won't affect the next command.

Reply:
Ah! Good one; I sometimes get confused when to `\` and when not in Makefiles